metricbeat-yml-script.py

```diff
@@ -78,6 +78,8 @@ def _add_modules():
         logger.error("Required at least one module")
         raise
     _enable_modules(modules)
+    if os.environ["CLOUD_METADATA"] == "true":
+        _enable_cloud_metadata(modules)
```

Comment:
Perhaps consider adding this part to `add_data_by_module` since it's already iterating over `modules`, and make `_enable_cloud_metadata` get only one module at a time?